var/spack/repos/builtin/packages/funwave/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack.package import *
+
+
+class Funwave(MakefilePackage):
+    """
+    What is FUNWAVE-TVD?
+    FUNWAVE-TVD is the Total Variation Diminishing (TVD) version of the fully nonlinear Boussinesq
+    wave model (FUNWAVE) developed by Shi et al. (2012). The FUNWAVE model was initially developed
+    by Kirby et al. (1998) based on Wei et al. (1995). The development of the present version was
+    motivated by recent needs for modeling of surfzone--cale optical properties in a Boussinesq
+    model framework, and modeling of Tsunami waves in both a global/coastal scale for prediction
+    of coastal inundation and a basin scale for wave propagation.
+    """
+
+    homepage = "https://fengyanshi.github.io/build/html/index.html"
+    url = "https://github.com/fengyanshi/FUNWAVE-TVD"
```

Comment:
This isn't being used since you're only providing git tags.  IF you were to use it, you'll want the path to a source archive file (e.g., `https://github.com/fengyanshi/FUNWAVE-TVD/archive/refs/tags/Version_3.6.tar.gz`).